src/components/Card.js

```diff
@@ -19,32 +18,28 @@ class Card {
     this._handleImageClick = handleImageClick;
     this._handleDeleteClick = handleDeleteClick;
     this._handleApiLike = handleLikeClick;
-  }
-
-  getCardElement() {
     this._cardElement = document
       .querySelector(this._cardSelector)
       .content.querySelector(".card")
       .cloneNode(true);
-
     this._cardTitleEl = this._cardElement.querySelector(".card__title");
     this._cardTitleEl.textContent = this._title;
-
+    this._cardLike = this._cardElement.querySelector(".card__button-like");
     this._cardImageEl = this._cardElement.querySelector(".card__image-card");
     this._cardImageEl.src = this._link;
     this._cardImageEl.alt = this._title;
     this._cardLikesEl = this._cardElement.querySelector(".card__button-count");
     this._cardLikesEl.textContent = this._getLikes();
     this._trashEl = this._cardElement.querySelector(".card__button-del");
+    this._cardLike = this._cardElement.querySelector(".card__button-like");
+  }
+
+  getCardElement() {
     if (this._cardOwnerId !== this._myId) {
       this._trashEl.classList.add("card__button-del-inactive");
     }
 
-    if (this.isLiked()) {
-      this._cardElement
-        .querySelector(".card__button-like")
-        .classList.add("card__button-like_active");
-    }
+    this.renderLikes();
 
     this._setEventListeners();
 
```

Comment:
```js

  _setEventListeners() {
    this._cardElement // handle a click on the like button
      .querySelector(".card__button-like")
      .addEventListener("click", () => {
```

You've defined `this._cardLike` in line 27. Please, use it here



```js

    this._cardLike.addEventListener("click", () => {

```

Reply:
Corrected